src/lib/clients/IOClients/apps/Builder.ts

```diff
@@ -14,13 +14,26 @@ export interface RequestParams {
   skipSemVerEnsure?: boolean
 }
 
+export interface BuilderHubRequestOptions {
+  params?: RequestParams
+  headers?: Record<string, any>
+}
+
 export interface BuildResult {
   availability?: AvailabilityResponse
   code?: string
   message?: any
   timeNano?: number
 }
 
+export interface PrepareRequest {
+  zipFile?: Buffer
```

Comment:
Is zipFile optional?

Reply:
No, I saw that and forgot to change, sorry!